algorithms/gr-dij/javascript/dijkstra_algorithm.js

```diff
@@ -0,0 +1,52 @@
+/*
+* Dijkstra's single source shortest path algorithm in JavaScript.
+ */
+ const dijkstraAlgorithm = function (graph, start) {
+
+    //the distances from the start node to all other nodes
+    var distance = [];
+    //Initial distance is "Infinity"
+    for (var i = 0; i < graph.length; i++) distance[i] = Number.MAX_VALUE;
```

Comment:
Codacy found an issue: ['i' used outside of binding context.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6358206)